llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -15942,6 +15942,16 @@ static bool isHalf2OrBFloat2(Type *Ty) {
   return false;
 }
 
+static bool isHalf2(Type *Ty) {
```

Comment:
Can we make this `isV2F16` please? LLVM already has a couple of conventions for naming vector types, I don't think we need to add another one.